pkgs/development/libraries/wayland/default.nix

```diff
@@ -71,16 +78,16 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
-    libffi
     expat
     libxml2
-  ] ++ lib.optionals withDocumentation [
+  ] ++ lib.optional withLibraries libffi
```

Comment:
```suggestion
  ] ++ lib.optionals withLibraries [ libffi ]
```

Reply:
I've pushed a slightly modified variant to keep the style consistent.